components/builder-api/src/server/resources/projects.rs

```diff
@@ -131,6 +145,12 @@ fn create_project(req: HttpRequest,
         }
     };
 
+    let target = if body.target.is_empty() {
+        PackageTarget::from_str("x86_64-linux").unwrap()
+    } else {
+        PackageTarget::from_str(&body.target).unwrap()
```

Comment:
Should check for err instead of unwrap.  Also if we're going to use this code in several places would be good to make it a helper function.